linux-anvil-aarch64/Dockerfile

```diff
@@ -0,0 +1,79 @@
+FROM continuumio/miniconda3 as builder
+
+RUN conda install constructor
+
+ADD constructors /opt/constructors
+
+RUN cd /opt/constructors/aarch64 && \ 
+    constructor --platform=linux-aarch64 .
+
+FROM arm64v8/centos:7
+
+ADD qemu-aarch64-static /usr/bin/qemu-aarch64-static
```

Comment:
Can we `ADD https://github.com/multiarch/qemu-user-static/releases/download/${QEMU_STATIC_VERSION}/qemu-aarch64-static /usr/bin/qemu-aarch64-static` here?

Reply:
sadly no, since that does not add it as an executable

Reply:
and since you aren't running on arm/ppc you fail if you try to chmod +x without this.